runtime/src/dpe_crypto.rs

```diff
@@ -53,7 +53,7 @@ impl<'a> DpeCrypto<'a> {
             sha384,
             trng,
             ecc384,
-            hmac384,
+            hmac: hmac384,
```

Comment:
Just above we can rename `hmac384: &'a mut Hmac` to just `hmac: ...`

Reply:
Updated.